bugbug/repository.py

```diff
@@ -51,24 +51,39 @@ def __init__(
         desc,
         date,
         pushdate,
-        bug,
+        bug_id,
         backedoutby,
         author_email,
-        files,
-        file_copies,
         reviewers,
+        files=[],
+        file_copies={},
+        ignored=False,
     ):
         self.node = node
         self.author = author
+        self.bug_id = bug_id
         self.desc = desc
         self.date = date
         self.pushdate = pushdate
-        self.bug = bug
         self.backedoutby = backedoutby
+        self.ever_backedout = backedoutby != ""
         self.author_email = author_email
-        self.files = files
+        if files:
```

Comment:
Do we have this parameter only for tests? Could you remove it and call `set_files` in tests directly?